core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -72,6 +72,9 @@ package object config {
   private[spark] val EVENT_LOG_OVERWRITE =
     ConfigBuilder("spark.eventLog.overwrite").booleanConf.createWithDefault(false)
 
+  private[spark] val EVENT_LOG_CALLSITE_FORM =
+    ConfigBuilder("spark.eventLog.callsite").stringConf.createWithDefault("short")
```

Comment:
Do we need a default here? `short` is defined? Where is the test case?

Reply:
Not sure whether we should introduce a conf here. cc @rxin

Reply:
Ah yeah this should have been documented, that is a good point. I should have looked more carefully at the configs. The other eventLog configs aren't internal either. 

I agree this could also be a boolean, or simply handle anything but "short" or "long" as "short"